### ISSUE TITLE
Conan v0.21.0 generates invalid VS props file for VS 2017

### ISSUE THREAD

**Issue by dmitry-zakablukov** [conan-io/conan#1135]
Good day.

I have upgraded to conan 0.21.0 from 0.19.0 and now I can't build projects in VS 2017 any more.
The error message is:
```
conanbuildinfo.props(6,5): error MSB5016: The name "Conan.AapGenerator.Root" contains an invalid character ".".
```

I suppose this is because of the following section in props file:
```
  <PropertyGroup Label="Conan.RootDirs">
    <Conan.AapGenerator.Root>C:\Users\user\.conan\data\AapGenerator\1.1\passware\stable\package\5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9</Conan.AapGenerator.Root>
    <Conan.libxml2.Root>C:\Users\user\.conan\data\libxml2\2.9.4\passware\stable\package\05f648ec4d066b206769d6314e859fdd97a18f8d</Conan.libxml2.Root>
    <Conan.zlib.Root>C:\Users\user\.conan\data\zlib\1.2.11\passware\stable\package\2a0b7ed68eb56c5f3ad8a5651d2edee732783124</Conan.zlib.Root>
  </PropertyGroup>
```

Please, remove dots from XML key names.

Best regards, Dmitry.

**Comment by memsharded** [comment]
Hi Dmitry,

Sorry for that. We are preparing a minor release that will fix this asap.

**Comment by dmitry-zakablukov** [comment]
@memsharded , thank you for quick response!

**Comment by memsharded** [comment]
Just released 0.21.1! Please upgrade
